linker/src/lib.rs

```diff
@@ -57,7 +55,7 @@ pub fn link<T: FieldElement>(graph: PILGraph<T>) -> Result<PILFile<T>, Vec<Strin
                 // the lhs is `instr_flag { inputs, outputs }`
                 let lhs = SelectedExpressions {
                     selector: Some(from.flag),
-                    expressions: once(Expression::Number(to.operation.id))
+                    expressions: to.operation.id.map(Expression::Number).into_iter()
```

Comment:
No: I ended up going with the latter approach in the issue. The former would have required introducing a magic operation_id which would then get removed. I decided against it to avoid this magic code. I updated the test to use the single operation. Now `to.operation.id` is None.